autoscheduler/scraper/management/commands/scrape_depts.py

```diff
@@ -1,23 +1,29 @@
 # Called to execute the beginning of scaping courses
+import time
+import datetime
+
 from django.core.management import base
 from scraper.banner_requests import BannerRequests
 from scraper.models import Department
 
-import time
-import datetime
-
 def parse_departments(json, term):
     """ Takes json list of departments and saves them as Department objects """
 
     for dept in json:
-        dept = Department(id=dept["code"]+term, code=dept["code"], description=dept["description"], term=term) 
+        dept = Department(
+          id=dept["code"]+term,
+          code=dept["code"],
+          description=dept["description"],
+          term=term)
         dept.save()
 
 def scrape_departments(term):
-        request = BannerRequests(term)
-        json = request.get_departments()
-        print(json)
-        parse_departments(json, term)
+    """ Takes term input and collects json object of departments """
+
+    request = BannerRequests(term)
+    json = request.get_departments()
+    print(json)
```

Comment:
Need to remove the print statement, as it'll clutter up the output when we run it